_includes/table-of-contents.html

```diff
@@ -17,7 +17,7 @@ <h3 class="table-of-contents__h3">
 
               <ul class="table-of-contents__ul">
                 {% for i in item.sub-items %}
-                  <li class="table-of-contents__li">
+                  <li class="table-of-contents__li" aria-labelledby="{{i.name}}">
```

Comment:
`aria-labelledby` should provide the ID of an HTML element that labels the element.
This maybe means to be `aria-label`?

I'm surprised that ARIA treatment would be necessary here. The body of the `<li>` is an anchor tag, the body of which is `i.name` as plaintext. So I'd have expected screen readers to consider that plain text the readable label for the `<li>`.